src/Artsy/Analytics/trackingMiddleware.ts

```diff
@@ -43,13 +44,34 @@ export function trackingMiddleware(options: TrackingMiddlewareOptions = {}) {
           if (!foundExcludedPath) {
             logger.warn("Tracking PageView:", pathname)
 
-            analytics.page(
-              { path: pathname, referrer },
-              { integrations: { Marketo: false } }
-            )
+            const trackingData: {
+              path: string
+              referrer?: string
+              url: string
+            } = {
+              path: pathname,
+              url: sd.APP_URL + pathname,
+            }
+
+            // TODO: Remove after EXPERIMENTAL_APP_SHELL AB test ends.
+            if (getENV("EXPERIMENTAL_APP_SHELL")) {
+              trackingData.referrer = sd.APP_URL + referrer
+            }
+
+            // TODO: Remove after EXPERIMENTAL_APP_SHELL AB test ends.
+            if (
+              ["/collect", "/collections", "/collection/"].some(path =>
```

Comment:
So this basically ensures this for the collect suite of pages, which is always client-side routed.

Reply:
Yup